assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -121,11 +124,27 @@ const OrderSummaryItem = ( { cartItem } ) => {
 				<ProductImage image={ images.length ? images[ 0 ] : {} } />
 			</div>
 			<div className="wc-block-components-order-summary-item__description">
+				<span className="screen-reader-text" tabIndex={ 0 }>
+					{ sprintf(
+						/* translators: %d items of product %s */
```

Comment:
I think this translators comment could be made a bit more specific so translators have a better understanding of what these variables are. Something along these lines: `%1$d is the quantity of items and %2$s the name of the product`. How does it sound?

Reply:
Fixed as well.